configure.ac

```diff
@@ -5269,7 +5281,9 @@ int main(void)
 [ac_cv_buggy_getaddrinfo=no],
 [ac_cv_buggy_getaddrinfo=yes],
 [
-if test "${enable_ipv6+set}" = set; then
+if test "$ac_sys_system" = "Linux-android" || test "$ac_sys_system" = "iOS"; then
+  ac_cv_buggy_getaddrinfo="no"
+elif test "${enable_ipv6+set}" = set; then
```

Comment:
@freakboy3742: For both this and `ac_cv_aligned_required` above, I've taken the liberty of including iOS as well. This means the `--enable-ipv6` configure argument will no longer be required.